.github/workflows/vs16-clang-ci.yml

```diff
@@ -6,6 +6,12 @@ jobs:
   ci:
     name: windows-vs16
     runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - vs: VS16Win64
+          - vs: VS16Win32
```

Comment:
The `VS16Win32` value was a guess. Apparently [that was correct](https://github.com/lukka/run-cmake/blob/main/action.yml#L40) --- but it turns out [the build is still 64 bit](https://github.com/lemire/fast_float/runs/1403807091?check_suite_focus=true#step:3:21).

Reply:
Would you accept if I changed this to not use the`lukka/run-cmake` action?

Reply:
> Would you accept if I changed this to not use thelukka/run-cmake action?

We do not have to use `thelukka`. Maybe, however, we could reach out and report a problem if there is.

Reply:
https://github.com/lukka/run-cmake/issues/35

Reply:
@biojppm I am sure that the author will appreciate!